src/components/ViewList.js

```diff
@@ -70,6 +70,19 @@ const ViewList = () => {
     setFilterValue(e.target.value);
   };
 
+  const handleDelete = (e) => {
+    window.confirm('Would you like to delete your item?');
```

Comment:
Clicking "cancel" in the confirm still deletes the item from the database.  We'll need something here to conditionally delete based on the confirm response.